### PR TITLE
add changes Awesome books: with ES6

### PR DESCRIPTION
**Pull request by HanaSabih**
- Setup my project
- Create a directory called modules.
- Use ES6 syntax.
- Divide my code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
- Keep all functionalities of the app without errors.
- Refactor my methods and functions to arrow functions.
- Use let and const in the correct way.
- Display the current date below the navigation bar.
- use a library luxon
- Download the script and save it to my project.
- Import it as a module.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @HanaSabih,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Ensure you verify the CRR form before submitting it. This project is about Awesome books: ES6 but you submitted the form for Todo List Structure. Kindly verify your submissions.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import Storage, {
+  Card,
+  arrBooks,
+  bookTitle,
+  bookAuthor,
+} from './modules/storage.js';
+import Books from './modules/books.js';
+import { DateTime } from './modules/luxon.js';
+/* eslint-disable no-unused-vars */
```
Comment (rustam-tolipov) [inline index.js]: - [ ] Kindly remove the eslint-disabler as its use is not permitted. There are unused variables, bodyCont, tableContainer, and navbar, which may be causing this issue and can be removed if they are not necessary.


> Sometimes we encounter challenges when using ESLint. Unexpected errors may arise which are difficult to resolve quickly. If you find an issue that is functioning properly, but is still displaying an error message when uploaded to GitHub, I recommend disabling in `.eslintrc.json`.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+const navlist = document.querySelector('.navlist');
+const navadd = document.querySelector('.navadd');
+const navcontact = document.querySelector('.navcontact');
+
+const d = DateTime.local().toLocaleString(DateTime.DATETIME_FULL);
+
+document.querySelector('.date-display').innerHTML = d;
+
+let id = arrBooks.length;
+/* eslint-disable no-plusplus */
```
Comment (rustam-tolipov) [inline index.js]: - [ ] Kindly remove eslint disabler. You can solve this issue by changing i++ to i + 1.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (bookTitle.value === '' || bookAuthor.value === '') {
+    warningLabel.classList.remove('d-none');
+  } else {
+    warningLabel.classList.add('d-none');
+
+    const book = new Books(id++, bookTitle.value, bookAuthor.value);
```
Comment (rustam-tolipov) [inline index.js]:
```suggestion
    const book = new Books(id + 1, bookTitle.value, bookAuthor.value);
```

File: modules/books.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
```
Comment (rustam-tolipov) [inline modules/books.js]: - [ ] You are free to remove this as well. If I remove this disabler. I find no error in console.

File: modules/storage.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+/* eslint-disable max-classes-per-file */
```
Comment (rustam-tolipov) [inline modules/storage.js]: - [ ] In this project, students are expected to migrate classes into functions using ES6 arrow syntax. Please make the necessary changes accordingly.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (rustam-tolipov) [inline index.html:19]: [OPTIONAL] Your code design is impressive. It is my opinion that creating a new book would function well, yet I suggest that redirecting to the Books List page would be an ideal implementation.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @HanaSabih,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @HanaSabih 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights
✔️ Descriptive PR 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**